windows/win_firewall_rule.ps1

```diff
@@ -200,7 +200,7 @@ $fwsettings=@{}
 # Variabelise the arguments
 $params=Parse-Args $args;
 
-$enable=Get-Attr $params "enable" $null;
+$enable=Get-Attr $params "enable" "";
```

Comment:
If enable is a boolean it should default to False or at least null, not empty string, also this does not reflect what the docs state as the default.